app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,21 @@
+class PostsController < ApplicationControntroller
```

Comment:
- [ ] As required for the project, kindly ensure that all your views are working as right now, I noticed that only the  `users#index` can work perfectly but others are showing errors

Reply:
Thanks for the feedback @tresorsawasawa